src/dotnet/Chat.UI.Blazor/Components/LeaveChatConfirmationModal.razor

```diff
@@ -0,0 +1,33 @@
+@using Stl.Extensibility
+@implements IModalView<LeaveChatConfirmationModal.Model>
+@attribute [MatchFor(typeof(Model), typeof(IModalView))]
+
+<DialogFrame Title="Leave chat">
+    <Body>
+        <p>Are you sure you want to leave the chat?</p>
+    </Body>
+    <Buttons>
+        <div class="btn-group">
+            <Button Class="btn-cancel-outline" Click="OnCancel">Cancel</Button>
+            <Button Class="btn-error" Click="OnLeave">Leave</Button>
+        </div>
+    </Buttons>
+</DialogFrame>
+
+@code {
+    [Inject] Session Session { get; init; } = null!;
+    [Inject] UICommandRunner Commander { get; init; } = null!;
+    [CascadingParameter] BlazoredModalInstance ModalInstance { get; set; } = null!;
```

Comment:
Let's declare all parameters as public. Not sure why this works, but I recently run S&R to add "public" to all of such properties.